model/project_vars.go

```diff
@@ -157,6 +157,9 @@ func CopyProjectVars(oldProjectId, newProjectId string) error {
 	if err != nil {
 		return errors.Wrapf(err, "finding variables for project '%s'", oldProjectId)
 	}
+	if vars == nil {
+		vars = &ProjectVars{}
```

Comment:
Did you run into this case? No project should exist without vars so if no vars exists that seems like an error (although I guess we should be returning one here then).

Reply:
I ran into this nil pointer dereference while writing tests so I suppose that's just a matter of improving the test data -- but I noticed we do the same thing [here](https://github.com/evergreen-ci/evergreen/blob/6026a8f1f2aa5a4f24aaf89ff00c7dc124a71575/model/project_vars.go#L170-L172) so perhaps it has been encountered before?

Reply:
Ah interesting. I'm fine with just leaving it, just wanted to make sure it wasn't something you ran into in production.